src/KafkaFlow.Retry.UnitTests/KafkaFlow.Retry/Durable/RetryDurableMiddlewareTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace KafkaFlow.Retry.UnitTests.KafkaFlow.Retry.Durable
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
+    using FluentAssertions;
+    using global::KafkaFlow.Retry.Durable;
+    using global::KafkaFlow.Retry.Durable.Definitions;
+    using Moq;
+    using Xunit;
+
+    [ExcludeFromCodeCoverage]
+    public class RetryDurableMiddlewareTests
+    {
+        public static readonly IEnumerable<object[]> DataTest = new List<object[]>
+        {
+            new object[]
+            {
+                Mock.Of<ILogHandler>(),
+                null
+            }
+        };
+
+        [Theory]
+        [MemberData(nameof(DataTest))]
+        public void RetryDurableMiddleware_Ctor_Tests(
+            object logHandler,
```

Comment:
These two arguments should be the two concrete objects `ILogHandler` and `RetryDurableDefinition` respectively.